package/MDAnalysis/core/universe.py

```diff
@@ -370,14 +370,8 @@ def load_new(self, filename, format=None, in_memory=False, **kwargs):
             pass
 
         if reader is None:
-            # Check if we need to use ChainReader
-            if util.iterable(filename):
-                # Save the format and pass this to ChainReader
-                reader_format = 'CHAIN'
```

Comment:
I moved all special cases into the `get_reader_for` logic, rather than having some outside (CHAIN stuff)